utils/build-apple-framework.sh

```diff
@@ -40,7 +27,7 @@ function get_mac_deployment_target {
 # Build host hermes compiler for internal bytecode
 function build_host_hermesc {
   cmake -S . -B build_host_hermesc
-  cmake --build ./build_host_hermesc --target hermesc
+  cmake --build ./build_host_hermesc --target hermesc -j ${NUM_CORES}
```

Comment:
That is what the old code did, but I think it's better to have just a single build system that we use everywhere, to avoid regressing one or the other.